index.html

```diff
@@ -29,14 +38,171 @@ <h1 class="bio-header">I'm Alexandra. Glad to see you!</h1>
+         <ul class="card-languages">
+            <li class="card-language">HTML</li>
+            <li class="card-language">CSS</li>
+            <li class="card-language">Javascript</li>
+            
+         </ul>
+         <button class="card-btn">See Project</button>
+    </div>
+
+
+    <div class="card">
+        <div class="card-img">
+            <img src="img/work-3.png">
+        </div>
+        <div class="card-name"><h2>Tonic</h2></div>
+        <ul class="card-props">
+            <li  class="card-prop"><a href="#">Canopy</a></li>
+                    <li class="card-prop"><a><img src="img/Counter.png"></a></li>
+                    <li class="card-prop"><a href="#">Back End Dev</a></li>
+                    <li class="card-prop"><a><img src="img/Counter.png"></a></li>
+                    <li  class="card-prop"><a href="#">Canopy</a></li>
+        </ul>
+        <p class="card-text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+         <ul class="card-languages">
+            <li class="card-language">HTML</li>
+            <li class="card-language">CSS</li>
+            <li class="card-language">Javascript</li>
+            
+         </ul>
+         <button class="card-btn">See Project</button>
+    </div>
+
+
+    <div class="card">
+        <div class="card-img">
+            <img src="img/work-3.png">
+        </div>
+        <div class="card-name"><h2>Tonic</h2></div>
+        <ul class="card-props">
+            <li  class="card-prop"><a href="#">Canopy</a></li>
+                    <li class="card-prop"><a><img src="img/Counter.png"></a></li>
+                    <li class="card-prop"><a href="#">Back End Dev</a></li>
+                    <li class="card-prop"><a><img src="img/Counter.png"></a></li>
+                    <li  class="card-prop"><a href="#">Canopy</a></li>
+        </ul>
+        <p class="card-text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+         <ul class="card-languages">
+            <li class="card-language">HTML</li>
+            <li class="card-language">CSS</li>
+            <li class="card-language">Javascript</li>
+            
+         </ul>
+         <button class="card-btn">See Project</button>
+    </div>
+</section>
+
+<section class="about-section">
+    <div class="content">
+        <div class="about-1">
     
+            <h2 class="about-heading">About Myself</h2>
+                    <p class="about-text">
+                        Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
+                    </p>
+                    <div class="connect">
+                        <p class="connect-text">lets connect</p>
```

Comment:
- [ ] It would be better if you use a heading`(hx)` tag to display the **lets connect** text as it's a subheading. This will make your code semantically correct and also improve site accessibility.